### PR TITLE
Add <wbr> element

### PR DESCRIPTION
**Pull request by petele**

### CODE REVIEW

**Comment by foolip** [comment]
`<wbr>` shows up in https://github.com/mdn/browser-compat-data/issues/23796 so I'm not sure about the support in Edge. @petele what I would do here is try `<wbr>` in Edge 18 in BrowserStack to see if BCD is wrong. Do you have access to BrowserStack?

**Comment by petele** [comment]
I don't think I have BrowserStack access :(

**Comment by foolip** [comment]
OK, I copied the MDN example into https://software.hixie.ch/utilities/js/live-dom-viewer/?saved=12893 and tried that in Edge 15. It works. I doesn't work in IE11 however.

I think the best guess we can make is that support was added in Edge 12 since that's when most of the changes between IE and Edge happened.

**Comment by foolip** [comment]
I've sent https://github.com/mdn/browser-compat-data/pull/23804 to update BCD. That will change the Baseline low date, so we need to wait for that to reach us via a BCD release.

File: features/wbr.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <wbr>
-description: The `<wbr>` element TODO.
+description: The `<wbr>` HTML element represents a word break opportunity.
```
Comment (ddbeck) [inline features/wbr.yml:2]: Do you think it'd be excessive to elaborate on this?

```suggestion
description: The `<wbr>` HTML element represents a word break opportunity, to explicitly mark a place within a word where a line might be broken.
```

**Comment by foolip** [comment]
Still waiting for https://github.com/mdn/browser-compat-data/pull/23804...

**Comment by petele** [comment]
@foolip - can you add blocked to this one please?

**Comment by foolip** [comment]
The Edge version fix came in with https://github.com/web-platform-dx/web-features/pull/1482. @petele can you rebase and update dist?

**Comment by petele** [comment]
done

**Review by foolip — APPROVED** [review]
There's a minor discrepancy in the Safari version with caniuse, but too small to investigate I think.

**Pull request merged**